### PR TITLE
Disable multicore build for sphinx

### PR DESCRIPTION
**Pull request by nabobalis**
Not sure if I got the command correct.

Required due to upstream: https://github.com/pydata/pydata-sphinx-theme/issues/1643

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -99,8 +100,8 @@ extras =
     gallery: docs-gallery
 commands =
     pip freeze --all --no-input
-    gallery: sphinx-build -j auto --color -W --keep-going -b html -d _build/.doctrees . _build/html {posargs}
-    !gallery: sphinx-build -j auto --color -W --keep-going -b html -d _build/.doctrees -D plot_gallery=0 . _build/html {posargs}
+    gallery: SPHINX_COMMAND {posargs}
+    !gallery: SPHINX_COMMAND -D plot_gallery=0 {posargs}
```
Comment (nabobalis) [inline tox.ini:104]:
```suggestion
    !gallery: SPHINX_COMMAND -D plot_gallery=0 {posargs}
```

**Review by Cadair — APPROVED** [review]
Lgtm

File: tox.ini [hunk 2]
```diff
@@ -99,8 +100,8 @@ extras =
...
```
Comment (nabobalis) [inline tox.ini:104]:
```suggestion
    gallery: {env:SPHINX_COMMAND} {posargs}
    !gallery: {env:SPHINX_COMMAND} -D plot_gallery=0 {posargs}
```

File: tox.ini [hunk 3]
```diff
@@ -99,8 +100,8 @@ extras =
     gallery: docs-gallery
 commands =
     pip freeze --all --no-input
-    gallery: sphinx-build -j auto --color -W --keep-going -b html -d _build/.doctrees . _build/html {posargs}
-    !gallery: sphinx-build -j auto --color -W --keep-going -b html -d _build/.doctrees -D plot_gallery=0 . _build/html {posargs}
+    gallery: {env:SPHINX_COMMAND} {posargs}
+    !gallery: {env:SPHINX_COMMAND} -D plot_gallery=0 {posargs}
```
Comment (nabobalis) [inline tox.ini:104]: Need to check this will work before a merge since I add the flag at the end of the command.

**Comment by nabobalis** [comment]
Online test:

FAILED ../../.tox/py312-online/lib/python3.12/site-packages/sunpy/io/tests/test_filetools.py::test_write_file_fits[fname1] - FileNotFoundError: [Errno 2] No such file or directory: 'aia_171_image.fits'

Will be looked/fixed in another PR

**Pull request merged**